opentelemetry.html.md.erb

```diff
@@ -4,35 +4,36 @@ owner: Logging and Metrics
-to egress metrics.
+to egress traces, metrics, and logs.
 
-When the OpenTelemetry (OTel) Collector is deployed, the Loggregator Forwarder Agent forwards platform and application metrics
-to the OTel Collector, which then egresses metrics using one or more metric exporters that you configure.
+When the OpenTelemetry (OTel) Collector is deployed, the Loggregator Forwarder Agent forwards platform and application metrics to the OTel Collector, which then processes traces, metrics, and logs using one or more pipelines that you configure.
 
-<p class="note caution">
-<span class="note__title"><strong>Caution</strong></span>
-OpenTelemetry Collector is currently an experimental feature and should not be used in production systems.</p>
+# <a id='collector-config'></a> Collector configuration structure
 
-## <a id='metric-exporter-config'></a> Metric Exporter configuration
 
-The OTel Collector has a standard YAML-based file format for configuration. When deploying <%= vars.app_runtime_abbr %>
-with Otel Collector support, provide only the exporter configuration.
+The OTel Collector has a standard YAML-based file format for configuration. 
 
-<p class="note important">
-<span class="note__title"><strong>Important</strong></span>
-Specify only the configuration under the <code>exporters</code> section of the OpenTelemetry Collector configuration. It
-is not currently possible to configure Receivers or Processors.</p>
+The structure of <%= vars.app_runtime_abbr %> configuration file consists of three classes of pipeline components that access telemetry data:
+
+* [Exporters](#exporter-config)
+* [Receivers](#receiver-config)
+* [Processors](#processor-config)
 
-For example, to configure the OpenTelemetry Collector to egress metrics using OTLP over gRPC, you can provide the following
-minimal configuration. This will use the defaults for gRPC and TLS configuration.
+After each pipeline component is configured, add it in the `service.pipelines` [section](#service) of the Collector configuration file.
+
+## <a id='exporter-config'></a> Exporter configuration
+For example, to configure the OpenTelemetry Collector to egress metrics using OTLP over gRPC, you can provide the following minimal configuration. This will use the defaults for gRPC and TLS configuration.
 
 ```
 otlp:
   endpoint: 203.0.113.10:4317
 ```
 
-As the OTel Collector is typically deployed on all VMs, you must ensure that all VMs in your deployment are
-allowed to connect to the OTLP server endpoint.
+As the OTel Collector is deployed on all VMs in Cloud Foundry, you must ensure that all VMs in your deployment are allowed to connect to the OTLP server endpoint.
+
+<p class="note important">
+<span class="note__title"><strong>Important</strong></span>
+You can specify additional experimental exporters including <code>file</code>, <code>prometheusremotewrite</code>  and <code>splunkhec</code> after enabling the <code>experimental OpenTelemetry Collector features (beta)</code> checkbox.</p>
```

Comment:
We shouldn't refer to TAS checkboxes in the open source. Can you express the different exporters and their states in a different way?

I believe @mkocher previously suggested a table that lists the included component and the ga/beta/alpha status. That could be one way to go about this.

Reply:
added tables for processors and exporters